ci/sanity/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+echo "Install cstor-operators CRDs"
+
+kubectl apply -f ./deploy/crds
+
+echo "Apply RBAC rules"
+
+kubectl apply -f ./deploy/rbac.yaml
+
+echo "Install NDM-Operator"
+
+kubectl apply -f ./deploy/ndm-operator.yaml
+
+echo "Install cStor-Operators"
+
+kubectl apply -f ./deploy/cstor-operator.yaml
+
+echo "Install CSI"
+
+kubectl apply -f https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/csi-operator-ubuntu-18.04.yaml
```

Comment:
Can this also be maintained locally? to avoid the dependency on master vs branch cases.